src/coreclr/scripts/jitutil.py

```diff
@@ -119,15 +119,15 @@ def run_command(command_to_run, _cwd=None, _exit_on_fail=False, _output_file=Non
                     if proc.poll() is not None:
                         break
                     if output:
-                        output_str = output.strip().decode("utf-8")
+                        output_str = output.strip().decode("utf-8", errors='replace')
```

Comment:
Doesn't this mean utf-8 is not the right encoding to use here? It looks like Popen has a `text=True` arg we can set which will make `readline()` return a string instead of a bytes sequence, so then we won't need to deal with encoding ourselves:
https://docs.python.org/3/library/subprocess.html#subprocess.CompletedProcess.stdout
Maybe this will work better?

Reply:
I tried out on a sample file that contains unicode and seems it doesn't work smoothly. 

```
Exception in thread Thread-1:
Traceback (most recent call last):
  File "C:\Python39\lib\threading.py", line 973, in _bootstrap_inner
    self.run()
  File "C:\Python39\lib\threading.py", line 910, in run
    self._target(*self._args, **self._kwargs)
  File "C:\Python39\lib\subprocess.py", line 1479, in _readerthread
    buffer.append(fh.read())
  File "C:\Python39\lib\encodings\cp1252.py", line 23, in decode
    return codecs.charmap_decode(input,self.errors,decoding_table)[0]
```

There might be ways to fix it, but with `errors=replace`, it works fine even on the file I test. So I will just go ahead with current option. Also, it is not very common to hit this issue. It is just happening for certain benchmarks whose arguments contains unicode characters.